src/actions/list_objects_v2.rs

```diff
@@ -117,6 +117,19 @@ impl<'a> ListObjectsV2<'a> {
         self.query_mut().insert("prefix", prefix);
     }
 
+    /// A delimiter is a character that you use to group keys.
+    ///
+    /// See https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html#API_ListObjectsV2_RequestSyntax for more infos.
+    /// # Example
+    /// ```
+    /// # let bucket = rusty_s3::Bucket::new(url::Url::parse("http://rusty_s3/").unwrap(), rusty_s3::UrlStyle::Path, "doggo", "doggoland").unwrap();
+    /// let mut list = bucket.list_objects_v2(None);
+    /// list.delimiter("/");
```

Comment:
```suggestion
    /// list.with_delimiter("/");
```

Reply:
@paolobarbolini I fixed as you suggested.